src/vec_bcd_ppc.h

```diff
@@ -881,18 +881,18 @@ vec_bcddiv (vBCD_t a, vBCD_t b)
  * (of course they are different).
  *
  * But the BCD convert from/to Signed Quadword operations are a bit
- * heavier, running 37 and 23 cycles latency. These instructions
- * execute in the DFU and so are single issue. They also keep the DFU
- * pipeline busy (for 25 and 11 cycles) and block execution of the
- * next DFU operation for a while.  Still this is better than the
- * serial conversion examples described above.
+ * heavier, running 37 and 23 cycles latency on POWER9. These
+ * instructions execute in the DFU and so are single issue. They also
+ * keeps the DFU pipeline busy (for 25 and 11 cycles) and block
+ * execution of the next DFU operation for a while.  Still this is
+ * better than the serial conversion examples described above.
  *
  * But part of the value of PVECLIB is to provide support across
  * POWER7/8/9 and across compiler versions.
  * The convert instructions above are not supported in current
- * compilers with built-ins so PVECLIB should provide in-line
+ * compilers with built-ins so PVECLIB provides in-line
  * assembler implementations for these operations.
- * Now we should looking into better algorithms for implementing
+ * Now we need look into better algorithms to implementing
```

Comment:
I would've kept "for" here, instead of "to", but it's subjective.

Reply:
yes